app/fight/knightConfig.py

```diff
@@ -0,0 +1,32 @@
+class Knights:
+
+    def __init__(self, knights_char):
+        self.name = knights_char["name"]
+        self.power = knights_char["power"]
+        self.hp = knights_char["hp"]
+        self.armour = knights_char["armour"]
+        self.weapon = knights_char["weapon"]
+        self.potion = knights_char["potion"]
+        self.protection = 0
+        self.preparation_for_battle()
+
+    def preparation_for_battle(self):
+        self.power += self.weapon["power"]
+
+        for armour in self.armour:
+            self.protection += armour["protection"]
+        if self.potion is not None:
+            stats = ("protection", "power", "hp")
+            for stat in stats:
+                if stat in self.potion["effect"]:
+                    setattr(self,
+                            stat,
+                            getattr(self, stat) + self.potion["effect"][stat])
+
+    def order_def_winner(self, other):

```

Comment:
What does `def` mean?

Reply:
Sorry. It's a small mistake. I've already corrected.